### ISSUE TITLE
duplicate tasks show after deleting and recreating them in different automation policy

### ISSUE THREAD

**Issue by bbrendon** [wh1te909/tacticalrmm#554]
Tasks created by policy:
![image](https://user-images.githubusercontent.com/6364477/120836780-cc291c80-c51a-11eb-8ecf-e2b6a980fb44.png)

In my setup I have a global default policy and a client level policy. I removed the tasks from the policy on the client and created the task on the global policy. After that, some of the tasks are like you see in the screenshot. Some agents do not have duplicate tasks. Its very sporadic. I tried "Sync Policies" and also "Remove orphaned tasks". Neither made a difference.

Also, as I'm investigating this, I was looking in "Policy Overview" and the tasks assigned to workstation and servers do not match the global policy plus the client policy. Maybe I'm misunderstanding how that works but based on the behavior of the software, that was my expectation.

I'm running 0.6.13 on debian.
agents are 1.5.7. I haven't noticed any other versions.

**Comment by bbrendon** [comment]
Also, I have 5 of these celery errors in the logs in the last day or so. Not sure what they are about.

```
[2021-06-02 22:48:37,932: ERROR/ForkPoolWorker-2] Task automation.tasks.delete_policy_autotasks_task[2d68aed4-1bf8-4574-a878-44b6e94de18c] raised unexpected: DatabaseError('Save with update_fields did not affect any rows.')

```

**Comment by silversword411** [comment]
Are these policies from before 0.6.9? There was a bug with automation policies and weird stuff happening. Only deleting the entire policy and re-creating fixed them....thought you had said in Discord you already did that but.....

**Comment by bbrendon** [comment]
OOo. The policies for the sites were created before 0.6.9. the global policy i'm not sure.

**Comment by sadnub** [comment]
Thanks for your help on this! I was able to track down the issue to only tasks that run on check failure and only when agents were offline (or hadn't removed the old task since it was last removed)

I just pushed a fix. It will be in the next release.

**Comment by wh1te909** [comment]
released in 0.6.14

**Comment by bbrendon** [comment]
I hate to be the bearer of bad news but this doesn't seem to be resolved. At first it did but now I'm finding more and more again with duplicate TASKS. I haven't tested or enabled any checks. 

What I did was...
1. disable all automation policies.
2. wait about 20 minutes.
3. enable the global policy
4. duplicates appeared after about an hour. So far I have found at least 5.
